ios/TestApp/TestAppTests/TestAppTests.mm

```diff
@@ -0,0 +1,38 @@
+#import <XCTest/XCTest.h>
+#import <torch/script.h>
+
+@interface TestAppTests : XCTestCase
+
+@end
+
+@implementation TestAppTests {
+  torch::jit::script::Module _module;
+}
+
+- (void)setUp {
+  [super setUp];
+  NSString* modelPath = [[NSBundle bundleForClass:[self class]] pathForResource:@"model"
+                                                                         ofType:@"pt"];
+  XCTAssertTrue([[NSFileManager defaultManager] fileExistsAtPath:modelPath],
+                @"model.pt doesn't exist!");
+  auto qengines = at::globalContext().supportedQEngines();
+  if (std::find(qengines.begin(), qengines.end(), at::QEngine::QNNPACK) != qengines.end()) {
+    at::globalContext().setQEngine(at::QEngine::QNNPACK);
+  }
+  torch::autograd::AutoGradMode guard(false);
+  _module = torch::jit::load(std::string(modelPath.UTF8String));
+  _module.eval();
+}
+
+- (void)tearDown {
+  [super tearDown];
+}
```

Comment:
nit: not needed?

Reply:
better to keep it, might become useful in the future when more test are added.